spec/lib/danger/commands/init_helpers/interviewer_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rspec'
+require 'danger/commands/init_helpers/interviewer'
+
+RSpec.describe Danger::Interviewer do
+  let(:cork) { double('cork') }
+  let(:interviewer) { Danger::Interviewer.new(cork) }
+
+  describe '#link' do
+    before do
+      allow(interviewer).to receive(:say)
+    end
+
+    it 'link URL is decorated' do
+      interviewer.link('http://danger.systems/')
```

Comment:
Ah cool, what do you think about remove the `let` and `before` being explicit in place?

```ruby
require 'danger/commands/init_helpers/interviewer'

RSpec.describe Danger::Interviewer do
  describe '#link' do
    it 'link URL is decorated' do
      cork = double('cork')
      interviewer = Danger::Interviewer.new(cork)
      allow(interviewer).to receive(:say)

      interviewer.link('http://danger.systems/')

      expect(interviewer).to have_received(:say).with(" -> \e[4mhttp://danger.systems/\e[0m\n")
    end
  end
end
```